common/interfaces/flash_interface/flash_api.h

```diff
@@ -297,6 +297,20 @@ int update_challenge_flash(const char *name,
                            const uint8_t target[SHA256_SIZE],
                            const uint8_t random_number[POW_RAND_NUMBER_SIZE]);
 
+/**
+ * The function sets the wallet locked status and updates the locked card
+ * number(1-4).
+ *
+ * @param wallet_id The wallet_id is used to determine which wallet is locked.
+ * If that wallet is found, it's status is updated to locked
```

Comment:
Please update this comment. We are using wallet_name as key

Reply:
updated

Reply:
I think doxygen comments are duplicated for some reason, please check and fix

Reply:
updated